src/services/__tests__/helpers.spec.ts

```diff
@@ -113,8 +113,8 @@ describe('helpers', () => {
       expect(willNextExerciseUnlock(undefined, SCORE_THRESHOLD_UNLOCK + 1)).toBeTruthy()
     })
 
-    it('should unlock if exercise is done good enough for the scond time', () => {
-      expect(willNextExerciseUnlock(SCORE_THRESHOLD_UNLOCK, SCORE_THRESHOLD_UNLOCK + 1)).toBeTruthy()
+    it('should unlock if exercise is done good enough for the second time', () => {
+      expect(willNextExerciseUnlock(SCORE_THRESHOLD_UNLOCK, SCORE_THRESHOLD_UNLOCK + 2)).toBeTruthy()
```

Comment:
Thanks for fixing these typos. Why have you changed this from ` + 1 ` to ` +  2`. It is not needed to change this, but it is fine.